docs/en_us/platform_api/source/shared/courses_response_values.rst

```diff
@@ -0,0 +1,223 @@
+
+.. _Image:
+
+image
+======
+
+The following ``image`` objects have identical response values.
+
+* ``image`` (parent: ``course``, ``course_runs``, ``video``)
+* ``logo_image`` (parent: ``organization``)
+* ``profile_image`` (parent: ``person``)
+
+The ``image`` object has the following response values.
+
+* src (string): The URL where the image is located.
+* description (string): A description of the image.
+* height (integer): The height of the image in pixels.
+* width (integer): The width of the image in pixels.
+
+.. _Organization:
+
+organization
+==============
+
+The following ``organization`` objects have identical response values.
+
+* ``owners`` (parent: ``results``)
+* ``sponsors`` (parent: ``results``)
+
+The ``organization`` object has the following response values.
+
+* key (string): The unique ID for the organization.
+* name (string): The name of the organization.
+* description (string): A description of the organization.
+* logo_image (array): See :ref:`Image`.
+* homepage_url (string): The URL of the organization's home page.
+
+.. _Person:
+
+person
+=========
+
+The following ``person`` objects have identical response values.
+
+* ``instructor`` (parent: ``course_runs``)
+* ``staff`` (parent: ``course_runs``)
+
+The ``person`` object has the following response values.
+
+* key (string): A unique identifier for the instructor or staff member.
+* name (string): The first and last name of the instructor or staff member.
+* title (string): The official title of the instructor or staff member.
+* bio (string): Biographical information about the instructor or staff member.
+* profile_image (array): See :ref:`Image`.
+
+.. _Prerequisites:
+
+Prerequisites
+==================
+
+Parent: ``results``
+
+Any courses a learner must complete before enrolling in the current course.
+
+* name (string): ---The name of the prerequisite course. This is in
+  strikethrough text in the Google doc. Is it to be deleted?---
+
+.. _Seats:
+
+Seats
+=========
+
+Parent: ``course_runs``
+
+* type (string): The course mode or modes that the course offers. Possible
+  values are ``audit``, ``verified``, or ``professional education``.
+* price (string): The cost in USD of a verified certificate, a professional
+  education certificate, or academic credit for the course.
+* currency (string): The currency in which the course accepts payment. This
+  value must be ``USD``.
+* upgrade_deadline (string): The deadline for learners to upgrade from the
+  audit track to the verified certificate track.
+* credit_provider (string): The institution that offers academic credit for
+  learners who pass the course.
+* credit_hours (integer): The number of credit hours that learners who pass the
+  course earn.
+
+.. _Subjects:
+
+Subjects
+=========
+
+Parent: ``results``
+
+Academic subjects that this course covers.
+
+* name (string): Name of a subject, such as "computer science" or "history".
+
+**Possible values:**
```

Comment:
I don't think we know what this list is yet.  Until these are defined enumerables I would prefer to not list them in an exhaustive format.  Maybe a few examples for context